components/autofill/src/sync/address/outgoing.rs

```diff
@@ -0,0 +1,256 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/.
+*/
+
+use crate::db::models::address::InternalAddress;
+use crate::db::schema::ADDRESS_COMMON_COLS;
+use crate::error::*;
+use crate::sync::common::*;
+use crate::sync::{
+    OutgoingChangeset, Payload, ProcessOutgoingRecordImpl, ServerTimestamp, SyncRecord,
+};
+use rusqlite::{Row, Transaction};
+use sync_guid::Guid as SyncGuid;
+
+const DATA_TABLE_NAME: &str = "addresses_data";
```

Comment:
I created these `const`s mainly because I misspelled a table name and didn't want to make the same mistake as I created more tests. But beyond that development benefit the difference is negligible IMO. So basically I guess I'm writing more words just to agree with your words. 😂